src/mgmt/store/operator_store.py

```diff
@@ -21,6 +21,7 @@
 
 import logging
 import inspect
+from common.constants import KIND
 
```

Comment:
Is this tied to kind cluster?

Reply:
This is a reference to the Kind field for crd objects
```
apiVersion: "zeta.com/v1"
 kind: Fwd
 metadata:
```